src/components/MarkdownInAdmonitions.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {ListItem, OrderedList, Text, UnorderedList} from '@chakra-ui/react';
+
+import Markdown from 'react-markdown';
+import RelativeLink from './RelativeLink';
+import {HighlightKeyTerms} from '@apollo/pedia';
+import {MarkdownCodeBlock} from '@apollo/chakra-helpers';
+
+const LIST_SPACING = 1;
+
+const NESTED_LIST_STYLES = {
+  [['ul', 'ol']]: {
+    mt: 3,
+    fontSize: 'md',
+    lineHeight: 'normal'
+  }
+};
+
+export const MarkdownInAdmonitions = ({children}) => {
+  return (
+    <Markdown
+      components={{
+        ul: props => (
+          <UnorderedList
+            spacing={LIST_SPACING}
+            sx={{
+              ...NESTED_LIST_STYLES,
+              marginStart: '2em',
+              ul: {
+                listStyleType: 'circle'
+              }
+            }}
+            {...props}
+          />
+        ),
+        ol: props => (
+          <OrderedList
+            spacing={LIST_SPACING}
+            sx={NESTED_LIST_STYLES}
+            {...props}
+          />
+        ),
+        li: ({children, ...props}) => (
+          <ListItem
+            sx={{
+              '>': {
+                ':not(:last-child)': {
+                  mb: 3
+                }
+              }
+            }}
+            {...props}
+          >
+            <HighlightKeyTerms>{children}</HighlightKeyTerms>
+          </ListItem>
+        ),
+        p: ({children}) => {
+          return (
+            <Text>
+              <HighlightKeyTerms>{children}</HighlightKeyTerms>
+            </Text>
+          );
```

Comment:
I tried an admonition with appendText in my local repo, and the appended text started  on a new line. Removing `<Text>` here removed the new line. Is that an acceptable fix?

```suggestion
        p: ({children}) => {
          return (
              <HighlightKeyTerms>{children}</HighlightKeyTerms>
          );
```

Reply:
Hmm, this messes up some other styling, e.g. with custom text for the `EnterpriseFeature` component. E.g., before:

<img width="711" alt="image" src="https://github.com/apollographql/docs/assets/23219998/c5299056-7c92-4711-b1d9-9d29a25efe54">


and after:

<img width="712" alt="image" src="https://github.com/apollographql/docs/assets/23219998/ddfdd1ee-4f06-4f9b-b1b2-a650f2c4aaaf">

Reply:
There's got to be a way to have both of them looking good though, let me look into it.